src/Constants.js

```diff
@@ -54,4 +55,4 @@ export const SPECIAL_ELEMENTS = {
 };
 
 // These elements are special because they cannot contain childNodes.
-export const SELF_CLOSING_ELEMENTS = {img: 1, input: 1};
+export const SELF_CLOSING_ELEMENTS = {iframe: 1, img: 1, input: 1};
```

Comment:
OK. I guess this didn't work. Let's remove it from here again. Then we'll just find another way to solve this..